airflow/jobs/scheduler_job_runner.py

```diff
@@ -1066,6 +1128,14 @@ def _run_scheduler_loop(self) -> None:
                         num_finished_events += self._process_executor_events(
                             executor=executor, session=session
                         )
+
+                for executor in self.job.executors:
+                    try:
+                        with create_session() as session:
+                            self._process_task_event_logs(executor._task_event_logs, session)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rohankokkulabito/airflow/pull/6#issuecomment-2251038846>#720bfc</a><span> - 07/25/2024, 05:35 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Private member accessed: '_task_event_logs'
</div>
<div id="fix">
<b>Fix:</b> Avoid accessing private member '_task_event_logs' directly.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1134,7 +1134,7 @@
                 executor=executor, session=session
             )

         for executor in self.job.executors:
             try:
                 with create_session() as session:
                     self._process_task_event_logs(executor.get_task_event_logs(), session)
             except Exception:
                 self.log.exception("Something went wrong when trying to save task event logs.")
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged